pkg/controller/plan/adapter/ovirt/destinationclient.go

```diff
@@ -94,3 +95,33 @@ func (r *DestinationClient) DeleteObject(object client.Object, vm *plan.VMStatus
 	}
 	return
 }
+
+func (r *DestinationClient) findPVCByCR(cr *v1beta1.OvirtVolumePopulator) (pvc *core.PersistentVolumeClaim, err error) {
+	pvcList := core.PersistentVolumeClaimList{}
+	err = r.Destination.Client.List(
+		context.TODO(),
+		&pvcList,
+		&client.ListOptions{
+			Namespace: r.Plan.Spec.TargetNamespace,
+			LabelSelector: labels.SelectorFromSet(map[string]string{
+				"migration": string(r.Plan.Status.Migration.ActiveSnapshot().Migration.UID),
+				"diskID":    cr.Spec.DiskID,
+			}),
+		})
+
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+
+	if len(pvcList.Items) == 0 {
+		err = liberr.New("PVC not found", "diskID", cr.Spec.DiskID)
```

Comment:
maybe k8serr.NewNotFound, but we don't really need the specific error in this case